packages/vuetify/src/components/VOtpInput/VOtpInput.sass

```diff
@@ -0,0 +1,37 @@
+// Imports
+@import './_variables.scss'
+
+.v-otp-input
+  input
+    text-align: center
+  .v-input
+    margin: 0
+    padding: 0
+    .v-input__control
+      .v-input__slot
+        background: #ffffff
+    &.v-otp-input--plain
+      fieldset
+        display: none
+    input[type=number]::-webkit-outer-spin-button,
+    input[type=number]::-webkit-inner-spin-button
+      -webkit-appearance: none
+      margin: 0
+    input[type=number]
+      -moz-appearance: textfield
+  .theme--dark
+    .v-input__control
+      .v-input__slot
+        background: #303030
```

Comment:
Use the theme mixin for colors, this would be `map-get($material, 'bg-color')`

Reply:
Thank man - pushed a fix.